upsertGitHubTag/deployment/index.js

```diff
@@ -252,6 +254,27 @@ function processEvent(event, callback) {
         githubEventType +
         " from GitHub.",
     });
+    return;
+  }
+  // If bucket name is not null (had to put this for the integration test)
+  if (process.env.BUCKET_NAME) {
+    // Send payload to s3
+    const uploadDate = new Date();
+    const repository = body.repository.full_name;
+    const bucketPath = `${uploadDate.getDay()}-${uploadDate.getMonth()}-${uploadDate.getFullYear()}/${repository}/${deliveryId}`; //formats path to DD-MM-YYYY/repository/deliveryid
```

Comment:
`DD-MM-YYYY` a little odd, is there a reason for this? 
Otherwise, suggest `YYYY-MM-DD`